src/components/pages/vision/introduction.tsx

```diff
@@ -52,20 +67,42 @@ const Introduction: React.FC = () => {
           Vision
         </h1>
       </div>
-      <div className="w-full mx-auto md:max-w-md">
+      <div className="w-full ">
         {sectionCopy.map((copy, i) => {
           return (
-            <div key={i} className="mt-20 sm:mt-30">
-              <h2 className="mb-10 text-3xl font-bold sm:text-4xl sm:mb-14">
-                {copy.title}
-              </h2>
-              {copy.descriptions.map((description, index) => {
-                return (
-                  <p className="mt-8" key={index}>
-                    {description}
-                  </p>
-                )
-              })}
+            <div
+              key={i}
+              className="grid grid-cols-1 gap-8 mt-20 sm:mt-40 sm:gap-20 md:grid-cols-2"
+            >
+              <div className={i === 1 ? 'sm:order-last' : ''}>
+                <h2 className="mb-10 text-3xl font-bold sm:text-4xl sm:mb-14">
+                  {copy.title}
+                </h2>
+                {copy.descriptions.map((description, index) => {
+                  return (
+                    <p className="mt-8 text-xl" key={index}>
+                      {description}
+                    </p>
+                  )
+                })}
+              </div>
+
+              <div className={i === 1 ? '' : 'flex sm:justify-end'}>
```

Comment:
Same as above try to avoid using JS in className if CSS will suffice. Also if you do need logic in a className use the classNames dependency as seen in `wave.tsx`

Reply:
Tried using odd here but could not get to work.  The second row should have the quote box on left for larger screens and right on odd rows.